components/server/src/bitbucket-server/bitbucket-server-repository-provider.ts

```diff
@@ -102,6 +102,11 @@ export class BitbucketServerRepositoryProvider implements RepositoryProvider {
         return [];
     }
 
+    async getProviderRepos(user: User): Promise<ProviderRepository[]> {
+        // FIXME(janx): Not implemented yet
+        return [];
+    }
+
```

Comment:
Assume this FIXME is not intended to be addressed in this PR?

Reply:
You're right. 👍

As discussed earlier, I'd love to re-unify these "repository-handling" classes soon:
- [RepositoryProvider](https://github.com/gitpod-io/gitpod/blob/main/components/server/src/repohost/repository-provider.ts)
- [RepositoryService](https://github.com/gitpod-io/gitpod/blob/main/components/server/src/repohost/repo-service.ts)
- {GitHub,GitLab,Bitbucket}AppSupport (single method, no parent class, see e.g. [GitLabAppSupport](https://github.com/gitpod-io/gitpod/blob/main/components/server/ee/src/gitlab/gitlab-app-support.ts))

An in fact, this method `getProviderRepos` is actually already implemented in [BitbucketAppSupport](https://github.com/gitpod-io/gitpod/blob/main/components/server/ee/src/bitbucket/bitbucket-app-support.ts#L18-L78).

But I'd prefer leaving this refactoring for a follow-up Pull Request, so adding a FIXME for now.

Reply:
Ah, given that the EE vs not-EE split might still be relevant (due to licensing concerns), maybe it does make sense to keep two seperate classes (for example, unify the code into `RepositoryProvider` in not-EE and `RepositoryService` in EE -- in that case I think we'd need to move this `getProviderRepos` method into `RepositoryService`, which sort of makes sense because "get webhook-compatible repos" and "is repo webhook-compatible" and "install webhook on repo" are all related)

Reply:
Done.